### PR TITLE
fix: [CDS-38229]: added createpr icon

### PR DESCRIPTION
**Pull request by sahithikolichala**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Review by cjlee01 — APPROVED** [review]

**Review by tan-nhu — DISMISSED** [review]

File: packages/icons/src/create-pr.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="26" height="23" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M2.786 1.95c-.513 0-.929.428-.929.956v17.223c0 .529.416.957.929.957h20.428c.513 0 .929-.428.929-.957V7.69c0-.528.416-.957.928-.957.513 0 .929.429.929.957v12.44c0 1.585-1.247 2.87-2.786 2.87H2.786C1.247 23 0 21.715 0 20.13V2.905C0 1.321 1.247.036 2.786.036h16.25c.512 0 .928.428.928.956 0 .529-.415.957-.928.957H2.786zm3.882 5.192a.911.911 0 011.293-.236l4.371 3.154c.66.476.66 1.483 0 1.96l-4.37 3.152a.911.911 0 01-1.294-.235.975.975 0 01.228-1.332l3.556-2.566-3.556-2.565a.975.975 0 01-.228-1.332zm8.189 6.29c-.513 0-.929.428-.929.956 0 .529.416.957.929.957H19.5c.512 0 .928-.428.928-.957 0-.528-.416-.957-.928-.957h-4.643z" fill="#0278D5"/></svg>
```
Comment (abhinavrastogi-harness) [inline packages/icons/src/create-pr.svg:1]: pls remove height/width and use viewbox instead. pls replace hex fill color with `currentColor`.
 
 see SVG guidelines here: https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

**Review by abhinavrastogi-harness — CHANGES_REQUESTED** [review]

**Comment by sahithikolichala** [comment]
@abhinavrastogi-harness  this image is copied from figma as svg

**Comment by sahithikolichala** [comment]
@abhinavrastogi-harness  updated with designer suggested icon

File: packages/icons/src/create-pr.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.786 6.449c-.513 0-.929.428-.929.957v17.223c0 .529.416.957.929.957h20.428c.513 0 .929-.428.929-.957V12.19c0-.528.416-.957.928-.957.513 0 .929.429.929.957v12.44c0 1.585-1.247 2.87-2.786 2.87H5.786C4.247 27.5 3 26.215 3 24.63V7.405c0-1.586 1.247-2.87 2.786-2.87h16.25c.512 0 .928.428.928.956s-.415.957-.928.957H5.786zm3.882 5.192a.911.911 0 011.293-.235l4.371 3.153c.66.476.66 1.484 0 1.96l-4.37 3.153a.911.911 0 01-1.294-.235.975.975 0 01.228-1.333l3.556-2.565-3.556-2.565a.975.975 0 01-.228-1.333zm8.189 6.29c-.513 0-.929.429-.929.957s.416.957.929.957H22.5c.512 0 .928-.428.928-.957 0-.528-.416-.957-.928-.957h-4.643z" fill="#0278D5"/></svg>
```
Comment (abhinavrastogi-harness) [inline packages/icons/src/create-pr.svg:1]: please use `currentColor` for fill

**Pull request closed without merging**